qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -747,6 +760,7 @@ def on_load_finished(self, tab, ok):
         if idx == self.widget.currentIndex():
             self._update_window_title()
             tab.handle_auto_insert_mode(ok)
+        self._mark_dirty()
```

Comment:
Hmm, I guess this works - but I wonder if it wouldn't make more sense to have it in the tab objects rather than here - what do you think?